tests/test_bump_version.py

```diff
@@ -75,7 +75,7 @@ def test_bump_nextflow_version(datafiles, tmp_path):
     # Check .github/workflows/ci.yml
     with open(new_pipeline_obj._fp(".github/workflows/ci.yml")) as fh:
         ci_yaml = yaml.safe_load(fh)
-    assert ci_yaml["jobs"]["test"]["strategy"]["matrix"]["include"][0]["NXF_VER"] == version
+    assert ci_yaml["jobs"]["test"]["strategy"]["matrix"]["NXF_VER"][0] == version
```

Comment:
I feel like the actual code for `nf-core bump versions` should also need updating, no? Or do the existing regexes still work somehow?

Reply:
:facepalm: Yes it does. See d52a173

Reply:
Correct 👍🏻 The installation log is very noisy, just 100s of lines of download progress which is not informative in a non-interactive log.

Aim is to keep the CI workflows as clean as possible, so would be good to move it into the action and have it set by default.

Reply:
Done. See 0bab55b

Reply:
Great! Still needs implementing upstream though, right? https://github.com/nf-core/setup-nextflow/issues/1